src/routes/user.router.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const express = require('express');
```

Comment:
Consider destructuring 'Router' from 'express' to shorten the code and improve readability. Example: const { Router } = require('express');